addons/account/views/account_cash_rounding_view.xml

```diff
@@ -24,12 +24,12 @@
                         </div>
                         <group>
                             <field name="rounding"/>
-                            <field name="strategy"/>
+                            <field name="strategy" class="w-25"/>
                             <field name="profit_account_id" options="{'no_create': True}" attrs="{'invisible': [('strategy', '!=', 'add_invoice_line')], 'required': [('strategy', '=', 'add_invoice_line')]}"
                                 groups="account.group_account_invoice,account.group_account_readonly" domain="[('account_type', 'not in', ('asset_receivable', 'liability_payable'))]"/>
                             <field name="loss_account_id" options="{'no_create': True}" attrs="{'invisible': [('strategy', '!=', 'add_invoice_line')], 'required': [('strategy', '=', 'add_invoice_line')]}"
                                 groups="account.group_account_invoice,account.group_account_readonly" domain="[('account_type', 'not in', ('asset_receivable', 'liability_payable'))]"/>
-                            <field name="rounding_method"/>
+                            <field name="rounding_method" class="w-25"/>
                         </group>
```

Comment:
Instead you can use a second group in the first one. 
You will have something more Odooish. 
Ex:
![image](https://user-images.githubusercontent.com/6582885/194007669-28312b62-05f1-4520-943f-9b50ee8f8bbf.png)